sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -20,6 +20,8 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
+#include "mosquitto.h"
```

Comment:
Why here??

Reply:
Thanks for asking. I double checked and this is no longer needed here.
Before #8ccc3ac5d6db61e74ecb106e7ca8549728ad7a83, there was a reference in `az_mqtt5_rpc.h` to `MOSQ_ERR_SUCCESS`, which comes from `mosquitto.h`.
Since it's no longer needed I am removing it from this PR.